includes/apple-exporter/components/class-table.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Contains a component class representing a table.
+ *
+ * @package Apple_News
+ * @since 1.4.0
+ */
+
+namespace Apple_Exporter\Components;
+
+/**
+ * A component class representing a table.
+ *
+ * @since 1.4.0
+ */
+class Table extends Component {
+
+	/**
+	 * Look for node matches for this component.
+	 *
+	 * @param \DOMNode $node The node to inspect for matches.
+	 *
+	 * @access public
+	 * @return \DOMNode|null The DOMNode if there is a match, null if not.
+	 */
+	public static function node_matches( $node ) {
+
```

Comment:
Not a big deal, however, some phpcs config complain about this empty line

Reply:
Interesting. It looks weird to me if it's not spaced out like this, but that's certainly a personal preference. We're using a break like this in other places in the code. I've got a separate ticket to bring this entire repo up to modern wpcs coding standards, so if the config complains at that time, I'll fix it globally. Thanks!